pkgs/development/libraries/science/math/cudnn/releases.nix

```diff
@@ -155,17 +155,31 @@
     hash = "sha256-l2xMunIzyXrnQAavq1Fyl2MAukD1slCiH4z3H1nJ920=";
   }
   {
-    version = "8.8.0.121";
+    version = "8.8.1.3";
     minCudaVersion = "11.0";
     maxCudaVersion = "11.8";
-    url = "https://developer.download.nvidia.com/compute/cudnn/redist/cudnn/linux-x86_64/cudnn-linux-x86_64-8.8.0.121_cuda11-archive.tar.xz";
-    hash = "sha256-YgRkdgdtG0VfsT+3izjTSWusr7/bsElPszkiQKBEZuo=";
+    url = "https://developer.download.nvidia.com/compute/cudnn/redist/cudnn/linux-x86_64/cudnn-linux-x86_64-8.8.1.3_cuda11-archive.tar.xz";
+    hash = "sha256-r3WEyuDMVSS1kT7wjCm6YVQRPGDrCjegWQqRtRWoqPk=";
   }
   {
-    version = "8.8.0.121";
+    version = "8.8.1.3";
     minCudaVersion = "12.0";
     maxCudaVersion = "12.0";
-    url = "https://developer.download.nvidia.com/compute/cudnn/redist/cudnn/linux-x86_64/cudnn-linux-x86_64-8.8.0.121_cuda12-archive.tar.xz";
-    hash = "sha256-oHkrZmyq9ZOp3UEwl5V4/Tp4Iw9EB2RcKVcA7456qvI=";
+    url = "https://developer.download.nvidia.com/compute/cudnn/redist/cudnn/linux-x86_64/cudnn-linux-x86_64-8.8.1.3_cuda12-archive.tar.xz";
```

Comment:
While you're in the area, do you think it would be a good idea to abstract this url to a function, like I did in the TensorRT code here: https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/libraries/science/math/tensorrt/extension.nix#L34

Reply:
There are different URLs after after 8.4 and we're working on moving over to vendoring the redistributable manifests instead (after they correctly list the each CUDA version), so I didn't want to change anything else.